src/constants.ts

```diff
@@ -239,3 +241,321 @@ export type STU3Resource =
+    'CompartmentDefinition',
+    'Composition',
+    'ConceptMap',
+    'Condition',
+    'Consent',
+    'Contract',
+    'Coverage',
+    'CoverageEligibilityRequest',
+    'CoverageEligibilityResponse',
+    'DetectedIssue',
+    'Device',
+    'DeviceDefinition',
+    'DeviceMetric',
+    'DeviceRequest',
+    'DeviceUseStatement',
+    'DiagnosticReport',
+    'DocumentManifest',
+    'DocumentReference',
+    'EffectEvidenceSynthesis',
+    'Encounter',
+    'Endpoint',
+    'EnrollmentRequest',
+    'EnrollmentResponse',
+    'EpisodeOfCare',
+    'EventDefinition',
+    'Evidence',
+    'EvidenceVariable',
+    'ExampleScenario',
+    'ExplanationOfBenefit',
+    'FamilyMemberHistory',
+    'Flag',
+    'Goal',
+    'GraphDefinition',
+    'Group',
+    'GuidanceResponse',
+    'HealthcareService',
+    'ImagingStudy',
+    'Immunization',
+    'ImmunizationEvaluation',
+    'ImmunizationRecommendation',
+    'ImplementationGuide',
+    'InsurancePlan',
+    'Invoice',
+    'Library',
+    'Linkage',
+    'List',
+    'Location',
+    'Measure',
+    'MeasureReport',
+    'Media',
+    'Medication',
+    'MedicationAdministration',
+    'MedicationDispense',
+    'MedicationKnowledge',
+    'MedicationRequest',
+    'MedicationStatement',
+    'MedicinalProduct',
+    'MedicinalProductAuthorization',
+    'MedicinalProductContraindication',
+    'MedicinalProductIndication',
+    'MedicinalProductIngredient',
+    'MedicinalProductOperation',
+    'MedicinalProductManufactured',
+    'MedicinalProductPackaged',
+    'MedicinalProductPharmaceutical',
+    'MedicinalProductUndesirableEffect',
+    'MessageDefinition',
+    'MessageHeader',
+    'MolecularSequence',
+    'NamingSystem',
+    'NutritionOrder',
+    'Observation',
+    'ObservationDefinition',
+    'OperationDefinition',
+    'OperationOutcome',
+    'Organization',
+    'OrganizationAffiliation',
+    'Parameters',
+    'Patient',
+    'PaymentNotice',
+    'PaymentReconciliation',
+    'Person',
+    'PlanDefinition',
+    'Practitioner',
+    'PractitionerRole',
+    'Procedure',
+    'Provenance',
+    'Questionnaire',
+    'QuestionnaireResponse',
+    'RelatedPerson',
+    'RequestGroup',
+    'ResearchDefinition',
+    'ResearchElementDefinition',
+    'ResearchStudy',
+    'ResearchSubject',
+    'RiskAssessment',
+    'RiskEvidenceSynthesis',
+    'Schedule',
+    'SearchParameter',
+    'ServiceRequest',
+    'Slot',
+    'Specimen',
+    'SpecimenDefinition',
+    'StructureDefinition',
+    'StructureMap',
+    'Subscription',
+    'Substance',
+    'SubstancePolymer',
+    'SubstanceProtein',
+    'SubstanceReferenceInformation',
+    'SubstanceSpecification',
+    'SubstanceSourceMaterial',
+    'SupplyDelivery',
+    'SupplyRequest',
+    'Task',
+    'TerminologyCapabilities',
+    'TestReport',
+    'TestScript',
+    'ValueSet',
+    'VerificationResult',
+    'VisionPrescription',
+];
+
+export const SUPPORTED_STU3_RESOURCES: STU3Resource[] = [

```

Comment:
I would change the names now to be 'BASE' not 'SUPPORTED' as the interface shouldn't be saying what the server supports or not

Reply:
fixed